apps/search/vercel.json

```
@@ -0,0 +1,7 @@
+{
+  "functions": {
+    "src/pages/api/webhooks/saleor/**/*": {
+      "memory": 256
+    },
```

```suggestion
    }
```

Probably the trailing slash broke the release